docs/data/toolpad/core/components/dashboard-layout/DashboardLayoutBasic.js

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import DashboardIcon from '@mui/icons-material/Dashboard';
+import ShoppingCartIcon from '@mui/icons-material/ShoppingCart';
+import BarChartIcon from '@mui/icons-material/BarChart';
+import DescriptionIcon from '@mui/icons-material/Description';
+import LayersIcon from '@mui/icons-material/Layers';
+import { AppProvider } from '@toolpad/core/AppProvider';
+import { DashboardLayout } from '@toolpad/core/DashboardLayout';
+import theme from '../theme';
```

Comment:
It don't think we should have relative imports for demos, it will break once exported to codesandbox.

Reply:
Hm ok just wanted to reuse the same theme in all demos, will see how else it can be done.